https://issues.redhat.com/browse/ACM-14572 (#7081)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

- Cluster health
- Personalized view of your resources

> * You can now enable virtual machine actions to list, view, and filter `VirtualMachine` resources. See link:../console/enable_vm_actions.adoc#enable-vm-actions[Enabling virtual machine actions (Technology Preview)].

Reviewer: This should highlight the following 2 things:

1. There's a new virtual machine view in the infrastructure menu. _(This is in addition to the search results, the search results didn't change from previous releases.)_
2. You can trigger actions on the Virtual machine resources. _(This was added for both, the new view and the search results.)_

How about this:
```
You now have a dedicated view for virtual machine resources in addition to the view from search results. You can trigger actions on these virtual machine resources. See link:../console/enable_vm_actions.adoc#enable-vm-actions[Enabling virtual machine actions (Technology Preview)].
```

> * You can now enable virtual machine actions to list, view, and filter `VirtualMachine` resources. See link:../console/enable_vm_actions.adoc#enable-vm-actions[Enabling virtual machine actions (Technology Preview)].

Author: Thanks @jlpadilla what are your thoughts about my suggestion? 

```suggestion
* You can now view virtual machine resources from the console and your search results. Configure actions for the virtual machine resources. See link:../console/enable_vm_actions.adoc#enable-vm-actions[Enabling virtual machine actions (Technology Preview)].
```

Reviewer: Yes, that's good. Maybe avoid repeating virtual machine with this change.
```Configure actions for the virtual machine resources.```
